symphony-bdk-bom/build.gradle

```diff
@@ -70,6 +70,8 @@ dependencies {
         api 'org.freemarker:freemarker:2.3.30'
         api 'org.reflections:reflections:0.9.12'
 
+        api 'com.github.wnameless.json:json-flattener:0.12.0'
```

Comment:
Thanks for the PR @lukaszplawny, as we already use Jackson in the BDK is there a way this could work with https://github.com/FasterXML/jackson-dataformats-text/tree/2.13/properties ?

Reply:
Thanks for your comment @symphony-youri - I will check it and get back to you.